priya/javascript/Assignment6/JsAssign6Q2.html

```diff
@@ -48,3 +48,4 @@ <h4>Enter 5 names here</h4>
         </script>
     </body>
 </html>
```

Comment:
need to accept names till the user presses cancel in prompt dialog box and not only 5